bumble/hfp.py

```diff
@@ -361,9 +444,12 @@ class HfIndicatorState:
     enabled: bool = False
 
 
-class HfProtocol:
-    """Implementation for the Hands-Free side of the Hands-Free profile.
-    Reference specification Hands-Free Profile v1.8"""
+class HfProtocol(pyee.EventEmitter):
```

Comment:
It might be useful to document in the class docstring the list of events emitted from HfProtocol

Reply:
Done. That's also what I want to do in static check, but it will become a complicated mechanism.